### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by huhaham** [davide-coccomini/Combining-EfficientNet-and-Vision-Transformers-for-Video-Deepfake-Detection#47]
Hey, Davide, when I was working with the DFDC dataset, I downloaded the validation set and test set of DFDC on the https://dfdc.ai/ like you, but I don't know why the request always fails and cannot be downloaded
![微信图片_20230520153129](https://github.com/davide-coccomini/Combining-EfficientNet-and-Vision-Transformers-for-Video-Deepfake-Detection/assets/129136034/edc9d63a-8585-43cb-8370-0925fd0d1a91)

**Comment by zxq99799** [comment]
I have the same problem, have you resolved it?

**Comment by huhaham** [comment]
> 我有同样的问题，你解决了吗？

Sorry, I still couldn't solve the problem and I'm constantly looking for a way to fix it. Did you use only the training set of the DFDC dataset? How to skip the validation set of the DFDC dataset to run successfully train.py?

**Comment by zxq99799** [comment]
You can first separate a part of the dfdc training set for verification and testing.

**Comment by huhaham** [comment]
> You can first separate a part of the dfdc training set for verification and testing.
Ok, thanks, I'll give it a try, and can you give me your contact information?

**Comment by huhaham** [comment]
> You can first separate a part of the dfdc training set for verification and testing.

Ok, thanks, I'll give it a try, and can you give me your contact information?

**Comment by zxq99799** [comment]
z765791052@163.com This is my email, you can contact me through this.

**Comment by davide-coccomini** [comment]
Hi, did you solved?

**Comment by huhaham** [comment]
> Hi, did you solved?

I'm glad you answered my confusion, but I also couldn't download DFDC's validation and test sets during this time

**Comment by davide-coccomini** [comment]
Unfortunately, it seems that the authors of DFDC are not making the dataset available from their website, we can't do anything but wait for them...

**Comment by huhaham** [comment]
> Unfortunately, it seems that the authors of DFDC are not making the dataset available from their website, we can't do anything but wait for them...

Ok, I got it, thank you very much
